### PR TITLE
asset: Rename option to options

### PR DESCRIPTION
**Pull request by samuael**
# PR Description

The Instrument type "option" is what was added to the assets list. But, the instrument name "options" is supported by other exchanges. So, I have created(amended) the custom method calls for Okx outgoing calls to convert "options" to "option"; that way we can use the String() method call to get the asset Item name for other exchanges instead of creating a conversion method for each.

Also, we can use either "option" or "options" in the config.json files or with the asset.New() method and the result will be same.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [x] go test ./... -race
- [x] golangci-lint run
- [x] Test X

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/okx/okx.go [hunk 1]
```diff
@@ -4297,7 +4293,7 @@ func (ok *Okx) GuessAssetTypeFromInstrumentID(instrumentID string) (asset.Item,
 	filter := strings.Split(instrumentID, currency.DashDelimiter)
 	switch {
 	case len(filter) >= 4:
-		return asset.Option, nil
+		return asset.Options, nil
```
Comment (shazbert) [inline exchanges/okx/okx.go:4296]: above you can probably use `strings.Count()` function.

**Review by shazbert — CHANGES_REQUESTED** [review]
Looks good to me just one minor adjustment.

**Review by gloriousCode — APPROVED** [review]
tACK! Thanks Samuael! Congrats on OKx being merged too :call_me_hand:

**Review by thrasher- — APPROVED** [review]
tACK, upgrades smoothly 👍 Nice stuff @samuael 
image

**Pull request merged**